ubiquibot-config-default.json

```
@@ -81,7 +81,31 @@
   ],
   "incentives": {
     "comment": {
-      "elements": {},
+      "elements": {
+        "h1": 0,
+        "h2": 0,
+        "h3": 0,
+        "h4": 0,
+        "h5": 0,
+        "h6": 0,
+        "a": 0,
+        "ul": 0,
+        "li": 0,
+        "p": 0,
+        "img": 0,
+        "code": 0,
+        "table": 0,
+        "td": 0,
+        "tr": 0,
+        "br": 0,
+        "blockquote": 0,
+        "em": 0,
+        "strong": 0,
+        "hr": 0,
+        "del": 0,
+        "pre": 0,
+        "ol": 0
+      },
```

```suggestion
        "ol": 0
        "b": 0
        "i": 0
      },
```

I'm unsure if these are valid on GitHub but the intent here is targeting bold and italics.

---

Looks like its actually `em` on GitHub?